opentelemetry-api/src/opentelemetry/trace/__init__.py

```diff
@@ -369,8 +401,9 @@ class Tracer:
 
     # Constant used to represent the current span being used as a parent.
     # This is the default behavior when creating spans.
-    CURRENT_SPAN = Span()
+    CURRENT_SPAN = DefaultSpan(INVALID_SPAN_CONTEXT)
```

Comment:
`CURRENT_SPAN` is a sentinel object -- it just signals that we should look up the current span. Why change it here?

Reply:
Right, I changed it since `Span()` can't be instantiated. Could change it to something else though.